src/XamlStyler.dotUltimate/src/dotnet/ReSharperPlugin.XamlStyler.dotUltimate/Options/XamlStylerOptionsPage.cs

```diff
@@ -0,0 +1,215 @@
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using JetBrains.Annotations;
+using JetBrains.Application.Settings;
+using JetBrains.Application.UI.Controls.FileSystem;
+using JetBrains.Application.UI.Options;
+using JetBrains.Application.UI.Options.OptionPages;
+using JetBrains.Application.UI.Options.OptionsDialog;
+using JetBrains.Application.UI.Options.OptionsDialog.SimpleOptions;
+using JetBrains.DataFlow;
+using JetBrains.IDE.UI;
+using JetBrains.IDE.UI.Extensions;
+using JetBrains.IDE.UI.Extensions.Properties;
+using JetBrains.IDE.UI.Extensions.Validation;
+using JetBrains.IDE.UI.Options;
+using JetBrains.Lifetimes;
+using JetBrains.ReSharper.Feature.Services.Resources;
+using JetBrains.Rider.Model.UIAutomation;
+using JetBrains.Util;
+using Xavalon.XamlStyler.Options;
+
+namespace ReSharperPlugin.XamlStyler.dotUltimate.Options
+{
+    [OptionsPage(PID, PageTitle, typeof(FeaturesEnvironmentOptionsThemedIcons.FormattingStyle), ParentId = ToolsPage.PID)]
+    public class XamlStylerOptionsPage : BeSimpleOptionsPage
```

Comment:
Controls used here are `BeControls`, meaning they are constructed as a model in C#, then piped into the IntelliJ IDEA frontend that is Rider.